app/views/users/index.html.erb

```diff
@@ -1,12 +1,15 @@
-<div class="container"> 
-<% @users.each do |user| %>
-  <%= link_to user_path(user.id) do %>
-    <div class ="usercontainer">
-  <li class= "user">
-  <%=user.name %>
-  </li>
-  <span class="counter"> Number of Posts: <%=user.post_counter %> </span>
-  </div>
+<div class="container">
+  <% @users.each do |user| %>
+    <%= link_to user_path(user.id) do %>
+      <div class="usercontainer">
+        <li class="user">
+          <%= user.name %>
+        </li>
+        <span class="counter"> Number of Posts: <%= user.post_counter %> </span>
+      </div>
+    <% end %>
+    
   <% end %>
-<% end %>
-</div>
\ No newline at end of file
+  
+ 
```

Comment:
As I can't select the concerned part I select just this area, when I click on the `See all Posts` button I can't see all the posts and each post is not clickable I can't access a post, when I click on it nothing happens, please kindly fix that as well 🙏🏾 


https://github.com/Newtayo/blog_app/assets/74501523/eaa3097d-e129-4a6b-ac2c-4b4f78682f1a

- [Optional] Also I think it would be nice if you centered the elements of every page to match the given wireframe, it would be very nice

<img width="640" alt="Screenshot 2023-06-17 at 19 12 03" src="https://github.com/Newtayo/blog_app/assets/74501523/ff30b749-29d2-4ce3-9c3a-644dd7b94b28">